Sources/MapboxCoreNavigation/Directions+RoutingProvider.swift

```diff
@@ -8,30 +8,30 @@ extension URLSessionDataTask: NavigationProviderRequest {
 }
 
 extension Directions: RoutingProvider {
-    
+
     @discardableResult public func calculateRoutes(options: RouteOptions, completionHandler: @escaping RouteCompletionHandler) -> NavigationProviderRequest? {
-        return calculate(options, completionHandler: completionHandler)
+        return calculateWithCache(options, completionHandler: completionHandler)
     }
-    
+
     @discardableResult public func calculateRoutes(options: MatchOptions, completionHandler: @escaping MatchCompletionHandler) -> NavigationProviderRequest? {
-        return calculate(options, completionHandler: completionHandler)
+        return calculateWithCache(options, completionHandler: completionHandler)
```

Comment:
Given this Release Doc deprecation of calculateWithCache, perhaps there is a better way/method/injection to provide this service.
https://github.com/mapbox/mapbox-navigation-ios/blob/main/CHANGELOG.md#location-tracking-2